_episodes/07-checking.md

```diff
@@ -52,7 +52,7 @@ Run `make` on its own to get a full list of commands.
 
 In order to use Jekyll and/or the checking script,
 you may need to install it and some other software.
-The [setup instructions]({{ page.root }}/setup/) explain what you need and how to get it.
+The [setup instructions]({{ page.root }}/setup#optional-jekyll-setup-for-lesson-development/) explain what you need and how to get it.
```

Comment:
The link should be

~~~
{{ page.root }}/setup#optional-jekyll-setup-for-lesson-development
~~~

or

~~~
{{ page.root }}/setup/#optional-jekyll-setup-for-lesson-development
~~~

Web browsers and servers will ignore the trailing slash, see [here](https://webmasters.googleblog.com/2010/04/to-slash-or-not-to-slash.html) which means that `{{ page.root }}/setup#optional-jekyll-setup-for-lesson-development/` is the same as `{{ page.root }}/setup#optional-jekyll-setup-for-lesson-development` but I think very strange to have a trailing slash after the anchor.